src/utils/constants.ts

```diff
@@ -36,3 +36,10 @@ export const CATEGORY_RISK_LOW = 2;
 export const CATEGORY_RISK_MEDIUM = 3;
 export const CATEGORY_RISK_HIGH = 4;
 export const CATEGORY_RISK_VERY_HIGH = 5;
+
+export const PROGRAMME_TYPE_MULTILATERAL = 1;
+export const PROGRAMME_TYPE_DOMESTIC = 2;
+
+export const PROJECT_STATUS_COMPLETED = 2;
+export const PROJECT_STATUS_ONGOING = 1;
+export const PROJECT_STATUS_PLANNED = 0;
```

Comment:
Let's use `satisfies` if this is an enum.
Also, we should separate constants on different file